libtenzir/builtins/contexts/geoip.cpp

```diff
@@ -0,0 +1,485 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2023 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <tenzir/arrow_table_slice.hpp>
+#include <tenzir/data.hpp>
+#include <tenzir/error.hpp>
+#include <tenzir/fbs/data.hpp>
+#include <tenzir/flatbuffer.hpp>
+#include <tenzir/fwd.hpp>
+#include <tenzir/plugin.hpp>
+#include <tenzir/series_builder.hpp>
+#include <tenzir/type.hpp>
+#include <tenzir/view.hpp>
+
+#include <fmt/format.h>
+
+#include <chrono>
+#include <cstdint>
+#include <maxminddb.h>
+#include <memory>
+#include <string>
+
+namespace tenzir::plugins::geoip {
+
+namespace {
+
+auto constexpr path_key = "db_path";
```

Comment:
Our options are usually spelled `--long-option`, not `--long_option`. Can you change this?